libeos-updater-util/flatpak.c

```diff
@@ -1683,12 +1683,12 @@ fetch_runtime_ref_for_source_ref (FlatpakInstallation *installation,
       if (g_error_matches (local_error, G_IO_ERROR, G_IO_ERROR_NOT_FOUND))
         {
           g_autofree gchar *formatted_source_ref = flatpak_ref_format_ref (source_ref);
-          g_debug ("Remote %s did not have a summary cache "
-                   "or %s was not found in it, assuming that "
-                   "%s does not have a runtime",
-                   source_ref_remote_name,
-                   formatted_source_ref,
-                   formatted_source_ref);
+          g_message ("Remote %s did not have a summary cache "
```

Comment:
And here. Basically, I’d want these to be really obvious in the systemd journal. (And if we ever get automated QA, they could potentially be caught by a test which checks for warning (or higher) entries in the journal.)

Reply:
I'm not really sure how I feel about warning. If we test this behaviour `g_warning` and tests don't mix :/